src/scenes/Question/scenes/Read/index.js

```diff
@@ -1,31 +1,39 @@
 import React, { Component, Fragment } from 'react'
 import PropTypes from 'prop-types'
 import { Link } from 'react-router-dom'
-import { DateTime } from 'luxon'
 import Helmet from 'react-helmet'
 
-import { getNode } from './queries'
+import Card from 'react-toolbox/lib/card/Card'
+import CardTitle from 'react-toolbox/lib/card/CardTitle'
+import CardText from 'react-toolbox/lib/card/CardText'
+import Avatar from 'react-toolbox/lib/avatar/Avatar'
+import Tooltip from 'react-toolbox/lib/tooltip/Tooltip'
+import Button from 'react-toolbox/lib/button/Button'
+
+import { compose } from 'react-apollo'
+import { getNode, createFlag } from './queries'
 
 import { flags, markdown } from 'services'
 
 import NotFound from 'scenes/NotFound'
 
 import Loading from 'components/Loading'
 
-import Card from 'react-toolbox/lib/card/Card'
-import CardTitle from 'react-toolbox/lib/card/CardTitle'
-import CardText from 'react-toolbox/lib/card/CardText'
-import Avatar from 'react-toolbox/lib/avatar/Avatar'
-import Tooltip from 'react-toolbox/lib/tooltip/Tooltip'
-import Button from 'react-toolbox/lib/button/Button'
-import IconMenu from 'react-toolbox/lib/menu/IconMenu'
-import MenuItem from 'react-toolbox/lib/menu/MenuItem'
+import Meta from './components/Meta'
+import OptionsMenu from './components/OptionsMenu'
 
 const TooltipAvatar = Tooltip()(Avatar)
 
 class Read extends Component {
+  flag (type) {
```

Comment:
If you implements it as an arrow function you won't need to bind it later:

```js
flag = type => {
  // ...
}
```

http://babeljs.io/docs/plugins/transform-class-properties/

Reply:
I know about this, but this is an experimental feature plugin in babel (Source: https://stackoverflow.com/a/31362350)
However, I am using create-react-app which doesn't allow edition of the babel config at the moment (But is on the [roadmap for version 2](https://github.com/facebook/create-react-app/issues/3815) with babel macros)

Ejecting from create-react-app just to allow this feature seems counter-intuitive..

What do you think ?

Reply:
I agree 👍